portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -42,12 +42,54 @@ definition {
 	}
 
 	@description = "Verify that a Paragraph Field can be deleted"
-	@ignore = "true"
 	@priority = "5"
 	test CanBeDeleted {
+		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131223 CanBeDeleted pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Paragraph");
+
+		FormFields.editParagraphBody(content = "Test");
+
+		Form.closeSidebar();
+
+		Form.addTextField(fieldPositionNumber = "1");
+
+		Form.editElementSetFieldLabel(fieldValue = "Text");
```

Comment:
@LucasAcacio, as per test steps, I believe you should not edit the field label. 
Could you please confirm if this line is needed?